diff --git a/api/manage.py b/api/manage.py
index 2c50c4082..e12abdb7f 100755
--- a/api/manage.py
+++ b/api/manage.py
@@ -1,15 +1,6 @@
 #!/usr/bin/env python3
 
-import warnings
-
 from funkwhale_api.main import main
 
-warnings.warn(
-    DeprecationWarning(
-        "the './manage.py' script has been deprecated, please use the 'funkwhale-manage' "
-        "entrypoint instead (e.g. 'funkwhale-manage migrate')"
-    )
-)
-
 if __name__ == "__main__":
     raise SystemExit(main())
